packages/plugins/EVM/src/state/TransactionFormatter/types.ts

```diff
@@ -31,5 +31,6 @@ export interface TransactionComputationContext {
 export interface TransactionDescriptor {
     compute: (
         context: TransactionContext<ChainId, TransactionParameter>,
+        txHash?: string,
```

Comment:
We can read it from the `context`. Learn more: packages/web3-shared/base/src/specs/index.ts